.gitignore

```diff
@@ -21,3 +21,11 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+*.xml
+*.sh
```

Comment:
This ignore is too broad. Ignoring all .sh files may leave files behind without being committed. Please be more specific.